src/tests/ut/visit/visit_test.hpp

```diff
@@ -1,26 +1,40 @@
 #pragma once
 
+#include "doctor.hpp"
+#include "patient.hpp"
 #include "visit.hpp"
 #include "gtest/gtest.h"
 
 class Doctor;
+class Patient;
```

Comment:
detail: having both includes for doctor.hpp and patient.hpp forward declaration may be redundant although not an error